cmd/http-server/http-server.go

```diff
@@ -26,26 +27,35 @@ func main() {
 		return
 	}
 
-	listen := fmt.Sprintf("0.0.0.0:%d", *port)
-	router := StartCatalystAPIRouter()
+	mc := &handlers.MistClient{
+		ApiUrl:          fmt.Sprintf("http://localhost:%d/api2", *mistPort),
+		TriggerCallback: fmt.Sprintf("http://localhost:%d/api/mist/trigger", *port),
+	}
+
+	listen := fmt.Sprintf("localhost:%d", *port)
```

Comment:
Ah! This change reverted the listen address to `localhost` 😢

Reply:
Fixed in https://github.com/livepeer/catalyst-api/pull/22